src/core/theme/utils/focus.ts

```diff
@@ -27,3 +31,25 @@ export const focus = ({
         }
       `;
 };
+
+export const absoluteFocus = css`
```

Comment:
I'd rename this to something else as we discussed earlier. Maybe offsetFocus or pseudoFocus or something along those lines. Let's discuss.

Reply:
We discussed this further and decided to not take actions on this at this time. Leaving it as it is now.

Reply:
I prefer this name after all since this focus style cannot be used without absolute positioning. A longer name e.g. absolutelyPositionedFocus might be more descriptive but is already a bit cumbersome. Two hard things...